Gemfile.lock

```diff
@@ -0,0 +1,82 @@
+PATH
```

Comment:
I include the Gemfile.lock because I've moved the build process to use Docker now, so using the same .lock file makes sense since that the dev environments are the same